### ISSUE TITLE
Hail cannot load several plink files in parallel

### ISSUE THREAD

**Issue by danking** [hail-is/hail#3975]
### Hail version:
`08224c6ab`
### What you did:
Tried to load a plink dataset that was split by chromosome.
```
files = [(
    f'gs://fc-9a7c5487-04c9-4182-b3ec-13de7f6b409b/genotype/ukb_cal_chr{i}_v2.bed',
    f'gs://fc-9a7c5487-04c9-4182-b3ec-13de7f6b409b/genotype/ukb_snp_chr{i}_v2.bim'
) for i in range(1,23)]
mts = [hl.import_plink(bed=f[0],bim=f[1],fam="gs://phenotype_31063/ukb31063.fam") for f in files]
mt = mts[0].union_rows(*mts[1:])
```
### What went wrong (all error messages here, including the full java stack trace):
It loaded each plink file serially rather than in parallel, thus wasting many cores of my cluster (and my time).

**Comment by tpoterba** [comment]
I'd propose closing this as a won't-fix. The PLINK format doesn't lend itself well to parallel import, and spending even an engineer-day supporting it in a way that's more efficient than what you wrote above doesn't seem worth it.

Thoughts?

**Comment by danking** [comment]
I think Caitlin wanted this at one point but then switched back to the BGENs. I'm not sure why we have plink files in the first place. I am OK with punting on this until someone provides a compelling use case.

**Comment by tpoterba** [comment]
I think if you've sharded PLINK files by chromosome, you've already done a bad thing. You should go upstream to before you did that bad thing
